Alloy/lib/alloy/controllers/BaseController.js

```diff
@@ -219,9 +350,35 @@ var Controller = function() {
 
 		/**
 		 * @method updateViews
-		 * Applies a set of properties to view elements associated with this controller
+		 * Applies a set of properties to view elements associated with this controller.
 		 * @param {Object} args An object whose keys are the IDs (in form '#id') of views to which the styles will be applied.
 		 * @since 1.4.0
+		 * #### Example
+		 * The following example updates the UI elements with IDs "container"
+		 * and "name", in one case assing
+
+		 * View-controller file: controllers/profile.js
+
+	var profile = Alloy.createController('profile');
+	profile.updateViews({
+	 "#container" : {
+			layout : "vertical"
```

Comment:
I'm not sure I'd use `layout` as an example for using `updateViews()`. The intent of this new feature was to set properties on repeated elements (e.g. tableview rows) rather than needing to have a controller for those child controllers. See the test/apps/testing/ALOY-362 for examples.